### ISSUE TITLE
USER ISSUE: Chat Bugged

### ISSUE THREAD

**Issue by ghost** [StrangeLoopGames/EcoIssues#7338]
**Version:** 0.7.3.3 beta

**Steps to Reproduce:**
Type in #General

**Expected behavior:**
I expected to see what I typed after pressing enter in the chat

**Actual behavior:**
Other players could see what I typed but I could not see what I typed

**Comment by Ph1l95** [comment]
Hi,

the players on my server have the same problem. 
Expected behavior:
All players can read the chat.

Actual behavior:
A week ago one player had this problem, a few days later another and now the 3rd player has the problem.
They can write and the other players can see the message except the 3 players with the problem.

**Comment by MarjieAVolk** [comment]
So, when someone can't see what they are typing - do they see a blank space where their message should be?  Or do they just not see the message at all?  Can they see messages from other people?

**Comment by Graloth** [comment]
I have started experiencing this, as have other players on my server.

Some additional information:
- Affected player can still write messages, but cannot see these (no empty line where message should be, it appears as if never posted)
- Other players can see affected player messages just fine.
- Affected player cannot see other player chat messages.

Tried following, in hope it would fix issue (none did):
- Restarting server
- Restarting client
- Reinstalling and resetting client
